demos/JWST/S1_nirx_template.ecf

```diff
@@ -25,6 +25,30 @@ skip_gain_scale     False
 #Pipeline stages parameters
 jump_rejection_threshold  4.0 #float, default is 4.0, CR sigma rejection threshold
 
+#Custom bias frames
+custom_bias         False
+superbias_file	   /path/to/custom/superbias/fits/file
+
+#Saturation
+update_sat_flags    False   #Wheter to update the saturation flags more aggressively
+expand_prev_group   False   #Expand saturation flags to previous group
+dq_sat_mode         percentile # options: [percentile, min, defined]
+dq_sat_percentile   50      # percentile of the entire time series to use to define the saturation mask (50=median)
+dq_sat_columns	    [[0, 0], [0,0], [0,0], [0,0], [0,0]]  #for dq_sat_mode = defined, user defined saturated columns 
+
+#Background subtraction
+grouplevel_bg 		True
+ncpu				6
+bg_y1 				6
+bg_y2 				26
+bg_y_width          [None, None] # bkg from y[bg_y1-1-bg_y_width[0]: bg_y1-1] and y[bg_y2 + 1, bg_y2 + 1 + bg_y_width[1]]. ignored if None, can be [None, 4] or [4, None]  
+bg_deg 				0 
+p3thresh			5
+verbose				True
+isplots 			1
+isplots_S3 			False  #calling a S3 function requires this separate input
+hide_plots          True
+
```

Comment:
All of these new ECF parameters will need to be documented in the docs/source/ecf.rst file

Reply:
added!